docs/contratos.html

```diff
@@ -0,0 +1,117 @@
+<!doctype html>
```

Comment:
Cambié el indentado a 2 espacios en el editor, pero aquí no se refleja dicho cambio

Reply:
Estás usando VSCode? Si es así, verifica en la parte inferior que diga el tipo de identado. al momento de clonar tu repo a mi me sale lo siguiente en la barra inferior de status:

![image](https://user-images.githubusercontent.com/43255681/91587498-78026380-e91c-11ea-8558-0f1cb16f0da4.png)
y al seleccionar una línea salen los arrows del identado por "tabs"
![image](https://user-images.githubusercontent.com/43255681/91588056-3625ed00-e91d-11ea-8f02-239db972abed.png)

Eso indica que el identado se hace usando tabulaciones o "Tabs" de espaciado 4, para cambiarlo dale click en el indicador que dice "Tab Size: 4", te aparecerá un dropdown en la parte superior diciendo que tipo de identado quieres usar, le das en "Spaces" y posteriormente te pedirá tamaño, a lo que debes dar 2.
Posteriormente utiliza un shortcut de comando `Ctrl + Shift + P` y busca la opción de reidentar o reident, ésto aplicará tu configuración de espaciado al archivo activo actual, guardas tu archivo y subes el commit. Para verificar el cambio en el identado deberá mostrarse así:

![image](https://user-images.githubusercontent.com/43255681/91587708-c152b300-e91c-11ea-9301-dd8d1637d61d.png)
y al seleccionar una línea deben salir los puntos de identado por espacio.
![image](https://user-images.githubusercontent.com/43255681/91588128-4e960780-e91d-11ea-8f71-d8b9efbf565e.png)

Espero haberme dado a entender, saludos!

Reply:
Hola, gracias por la ayuda. Sin embargo, si tengo así confirgurado el VSCode
![imagen](https://user-images.githubusercontent.com/69448883/91646926-3bb52d00-ea1a-11ea-91f1-652324882f43.png)

No sé por qué en mi editor de texto si me aparece correcto en indentado, pero al momento de actualizar el archivo en el repo, se almacenan todos los cambios(camelCase y lo del For en el Label) menos el indentado.

Reply:
Si ejecutaste el comando para reidentar? Como comenté, una cosa es como tienes configurado tu editor y otra que aplique la configuración al archivo actual. Es posible que abras un archivo con identado diferente, **el editor no aplicará tu configuración a menos que se lo indiques**.